### PR TITLE
[RM Ch08] List of requirements for guiding test case selection

### LINKED ISSUE #470: [RM Ch08] Requirements for test case selection and graduation
We need to define a set of requirements and criteria which guide the evaluation and selection process of test cases beyond the pure scope of a test case. Such requirements include for instance availability in open source, documentation, maturity, etc. Having a list of formal requirements in place helps to guide discussions about suitability of test cases. The OpenStack Interop WG has made very good experiences with a formal set of evaluation criteria.

### PR DESCRIPTION
**Pull request by gkunz**
This adds a list of requirements test cases and tools must meet to be
eligable for inclusion in the compliance program.

Closes #470

### CODE REVIEW

File: doc/ref_model/chapters/chapter08.md [hunk 1]
```diff
@@ -790,6 +790,33 @@ The following five test categories have been identified as minimal testing requi
 
 <p>Respective test cases can be found in the <a title="Annex for Chapter 8" href="https://github.com/cntt-n/CNTT/blob/master/doc/ref_model/chapters/chapter08-annex.md" target="_blank" rel="noopener">Annex for Chapter 8</a>.</p>
 
+
+### 8.7.10 Test Case Selection Requirements
+
+This section lists requirements test cases must fulfill to be eligable for
```
Comment (deebhatia) [inline doc/ref_model/chapters/chapter08.md]: `eligable` -> `eligible`

File: doc/ref_model/chapters/chapter08.md [hunk 2]
```diff
@@ -790,6 +790,33 @@ The following five test categories have been identified as minimal testing requi
 
 <p>Respective test cases can be found in the <a title="Annex for Chapter 8" href="https://github.com/cntt-n/CNTT/blob/master/doc/ref_model/chapters/chapter08-annex.md" target="_blank" rel="noopener">Annex for Chapter 8</a>.</p>
 
+
+### 8.7.10 Test Case Selection Requirements
```
Comment (deebhatia) [inline doc/ref_model/chapters/chapter08.md]: Add an anchor tag on heading as done for others

File: doc/ref_model/chapters/chapter08.md (hunk 1 above)
Comment (gkunz) [reply doc/ref_model/chapters/chapter08.md]: thanks, will fix.

File: doc/ref_model/chapters/chapter08.md (hunk 2 above)
Comment (gkunz) [reply doc/ref_model/chapters/chapter08.md]: thanks, will add this.

**Comment by rabiabdel** [comment]
I dont see @Gaoweitao being active on here. may i ask @deebhatia to approve as well as @markshostak instead?

**Pull request merged**